src/storage/src/hummock/file_cache/utils.rs

```diff
@@ -75,3 +75,33 @@ pub fn align_down<U: Unsigned>(align: U, v: U) -> U {
     debug_assert_pow2(align);
     v & !(align - U::from(1))
 }
+
+macro_rules! bpf_buffer_trace {
+    ($buf:expr, $span:expr) => {
+        #[cfg(feature = "bpf")]
+        {
+            if $buf.len() >= 8 {
+                use bytes::BufMut;
+
+                const BPF_BUFFER_TRACE_MAGIC: u64 = 0xdeadbeefdeadbeef;
+
+                (&mut $buf[0..8]).put_u64_le(BPF_BUFFER_TRACE_MAGIC);
+
+                // Uncomment the followings, modify the len limits and uncomment bpf code to pass
+                // more fields.
+
+                // if let Some(id) = $span.id() {
+                //     (&mut $buf[8..16]).put_u64_le(id.into_u64());
+                // }
+
+                // let tts = std::time::SystemTime::now()
+                //     .duration_since(std::time::UNIX_EPOCH)
+                //     .unwrap()
+                //     .as_nanos();
+                // (&mut $buf[16..32]).put_u128_le(tts);
+            }
+        }
```

Comment:
Why do we need to put BPF magic in the buf if the following codes are commented?

Reply:
These codes are initially for adapting to OpenTelemetry for tracing. But after experiments and trials, I found it is not that easy. So I abandon these codes for now. Keeping them or removing them are both okay for me.